intg/src/main/java/org/apache/atlas/AtlasErrorCode.java

```diff
@@ -178,6 +178,7 @@ public enum AtlasErrorCode {
     TYPEDEF_ATTR_DISPLAY_NAME_IS_REQUIRED(400, "ATLAS-400-00-09F", "displayName is required for typedef \"{0}\" attribute"),
     EMPTY_REQUEST(400, "ATLAS-400-00-100", "Empty Request or null, expects Map of List of RelatedObjects with term-id as key"),
     TYPEDEF_DISPLAY_NAME_IS_REQUIRED(400, "ATLAS-400-00-101", "displayName is required for typedef"),
+    IMPORT_INVALID_ZIP_ENTRY(400, "ATLAS-400-00-09F", "{0}: invalid zip entry. Reason: {1}"),
```

Comment:
We already have error with same code "ATLAS-400-00-09F" 
Please update code for IMPORT_INVALID_ZIP_ENTRY

Reply:
Fixed, thanks for catch